source/2019-12-01-countdown-to-the-new-year-ember-sortable.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Countdown to The New Year: Ember Sortable
+author: Melanie Sumner
+tags: 2019, Addons, DecEmber
+alias: 2019-12-01-countdown-to-the-new-year-ember-sortable.md
+responsive: true
+---
+
+<b>This is the first in our DecEmber series- <i>"Countdown to The New Year: 31 Days of Ember Addons"</i>.  We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!</b>
+
+## Day 1
+
+To kick things off, we're going to look at [ember-sortable](https://emberobserver.com/addons/ember-sortable). 
+
+### What It Does
+
+The `ember-sortable` addon provides sortable UI primitives for Ember apps- this is super useful in the sense that teams don't have to reinvent this very common pattern every time they need it for their own applications! 
+
+### Why I Like It
+
+While the ember-sortable addon has been around for some time now, providing a consistent way to add drag and drop functionality to an Ember app, it is the most recent additions that have made this addon especially worth paying attention to. [Yicheng Gong](https://github.com/ygongdev), a software engineer at LinkedIn, noticed that the addon was missing accessibility support. He took on the work via the [Adopted Ember Addons org](https://github.com/adopted-ember-addons/), and updated the addon to add the missing features! 
```

Comment:
```suggestion
While the ember-sortable addon has been around for some time now, providing a consistent way to add drag and drop functionality to an Ember app, it is the most recent additions that have made this addon especially worth paying attention to. [Yicheng Gong @ygongdev](https://github.com/ygongdev), a software engineer at LinkedIn, noticed that the addon was missing accessibility support. He took on the work via the [Adopted Ember Addons org](https://github.com/adopted-ember-addons/), and updated the addon to add the missing features! 
```

We've been using this style on the Ember Times, seems helpful for folks that are known by their handle or the other way around, known by their name. Feel free to ignore though!